
From: Christoph Hellwig <hch@lst.de>

Not exactly something we want modules to mess around with.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/namespace.c |    4 ----
 1 files changed, 4 deletions(-)

diff -puN fs/namespace.c~remove-set_fs_root-set_fs_pwd fs/namespace.c
--- 25/fs/namespace.c~remove-set_fs_root-set_fs_pwd	2004-09-21 01:57:22.561008024 -0700
+++ 25-akpm/fs/namespace.c	2004-09-21 01:57:22.565007416 -0700
@@ -1209,8 +1209,6 @@ void set_fs_root(struct fs_struct *fs, s
 	}
 }
 
-EXPORT_SYMBOL(set_fs_root);
-
 /*
  * Replace the fs->{pwdmnt,pwd} with {mnt,dentry}. Put the old values.
  * It can block. Requires the big lock held.
@@ -1234,8 +1232,6 @@ void set_fs_pwd(struct fs_struct *fs, st
 	}
 }
 
-EXPORT_SYMBOL(set_fs_pwd);
-
 static void chroot_fs_refs(struct nameidata *old_nd, struct nameidata *new_nd)
 {
 	struct task_struct *g, *p;
_
